src/core/thread/topology.hpp

```diff
@@ -862,6 +862,56 @@ class Child : public Neighbor, public IndirectSender::ChildInfo, public DataPoll
 
 #endif // #if OPENTHREAD_CONFIG_CHILD_SUPERVISION_ENABLE
 
+#if OPENTHREAD_CONFIG_TMF_PROXY_MLR_ENABLE
+    /**
+     * This method returns MLR state of an Ip6 multicast address.
+     *
+     * @param[in] aAddress  The Ip6 multicast address.
+     *
+     * @returns MLR state of the Ip6 multicast address.
+     *
+     */
+    MlrState GetAddressMlrState(const Ip6::Address &aAddress);
```

Comment:
nit: declare as `const`?
```suggestion
    MlrState GetAddressMlrState(const Ip6::Address &aAddress) const;
```